common/ts/prepare-task.ts

```diff
@@ -54,6 +54,9 @@ export default async function prepareTask(endpoint: Endpoint, rootPath: string)
 
   if (scannerMode === ScannerMode.CLI) {
     let actualSonarOpts = tl.getVariable('SONAR_SCANNER_OPTS');
+    if (actualSonarOpts == null) {
```

Comment:
No need to change it, but you can write short things like `let actualSonarOpts = tl.getVariable('SONAR_SCANNER_OPTS') || '';` for the same result

Reply:
I knew the syntax in C# but i was looking for something like this, thanks !